### ISSUE TITLE
Main navigation has redundant label, missing  heading

### ISSUE THREAD

**Issue by paulschreiber** [techmatters/terraso-web-client#467]
### Main navigation lacks heading
The main navigation area (`<nav>` with class="MuiContainer-root MuiContainer-maxWidthLg css-yfw1zj") is not preceded by a heading, meaning that screen reader users cannot effectively use heading navigation commands to locate this important section of the website.

Add a level 2 heading as the first element of the `<nav>`, with the text "Main Navigation". This heading can be positioned off-screen (be screen-reader-only). The following CSS declaration will create a class called “sr-only”, which will position any element to which it is applied off-screen:

### Main navigation has redundant label
The main menu is encompassed in a `<nav>` with id="main-navigation" and aria-label="Main Navigation". Since the word "Navigation" is contained in the value of the aria-label attribute, it gets communicated twice, once from the accessible name and a second time from the programmatic role of the landmark.

Update the aria-label on `<nav>` to "Primary" or "Site" or "Main" and remove the word "Navigation" to avoid redundancy.
